agent/agent.go

```diff
@@ -3,12 +3,16 @@ package agent
 import (
 	"context"
 	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"sync"
 	"time"
 
 	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/apimodels"
 	"github.com/evergreen-ci/evergreen/command"
+	"github.com/evergreen-ci/evergreen/hostinit"
```

Comment:
can we not have this dependency? move the script name constant to evergreen/globals.go?

Reply:
done